cfgov/unprocessed/apps/filing-instruction-guide/js/fig-init.js

```diff
@@ -1,5 +1,8 @@
+import { SecondaryNav } from '../../../js/organisms/SecondaryNav.js';
 import { init as figSidenavInit } from './fig-sidenav.js';
 import { init as figSearchInit } from './fig-search.js';
 
-figSidenavInit();
-figSearchInit();
+let secondaryNav = SecondaryNav.init()[0];
```

Comment:
I'll swap this for `const` to placate the linter.

Reply:
ty!